### ISSUE TITLE
how to test trained model

### ISSUE THREAD

**Issue by aimaaonline** [sherjilozair/char-rnn-tensorflow#118]
i could not get how to test the model i have trained using train.py
please mention command.
i am using ubuntu 16.04

**Comment by itdaniher** [comment]
I think you're looking for sample.py to generate text.

**Comment by aimaaonline** [comment]
Yeah please guide me how to test the trained model. I have trained it on
some data sets. But now i could not get how it will generate text or
predict word

**Comment by aimaaonline** [comment]
i got it.the only problem was, that inspite of python i used python3  because it is python 3.5 which is configured with tensorflow 1.0 +
 python3 sample.py --init_dir=output --start_text="The meaning of life is" --length=10

**Comment by sangjeedondrub** [comment]
```sh
 ⚡  char-rnn-tensorflow ⎇  [master]  ✘ python sample.py --prime "It is a beautiful night" -n 10000
```
